Kickstarter-iOS/Features/PledgePaymentMethods/Views/Cells/PostCampaignPledgeRewardsSummaryCell.swift

```diff
@@ -77,34 +77,42 @@ final class PostCampaignPledgeRewardsSummaryCell: UITableViewCell, ValueCell {
     _ = ([self.titleLabel, self.amountLabel], self.rootStackView)
       |> ksr_addArrangedSubviewsToStackView()
   }
+
+  override func layoutSubviews() {
+    super.layoutSubviews()
+    self.titleLabel.preferredMaxLayoutWidth = self.titleLabel.frame.size.width
+    super.layoutSubviews()
```

Comment:
Ugly little hack to make multi-line labels work properly in automatically sized cells.